app/assets/stylesheets/content/_forms.sass

```diff
@@ -57,6 +57,8 @@ $form--field-types: (text-field, text-area, select, check-box, radio-button, ran
     padding: 30px 20px
     background-color: $content-form-bg-color
     border: $content-form-border
+  &.-higher
+    min-height: 31em
```

Comment:
erm. no. 

could you please check if there is a method using the `z-index` for the tooltip instead of relying on the form container being high enough?

The tooltip should be used in other parts of the application in the future so this is not an acceptable solution.